### PR TITLE
[twentyonepilots.com] Add ruleset

### PR DESCRIPTION
**Pull request by bn4t**

### CODE REVIEW

**Comment by bn4t** [comment]
@J0WI Thank you for the heads up.

File: src/chrome/content/rules/store.twentyonepilots.com.xml [hunk 1]
```diff
@@ -1,8 +1,10 @@
 <!--
-  Main domain (twentyonepilots.com) unfortunately doesn't supports https
+	Invalid Certificate:
+	- twentyonepilots.com
```
Comment (J0WI) [inline src/chrome/content/rules/store.twentyonepilots.com.xml]: And also `www.twentyonepilots.com`

Comment (bn4t) [reply src/chrome/content/rules/store.twentyonepilots.com.xml]: Right, forgot that. Thanks for mentioning it.

**Comment by J0WI** [comment]
Triggering Travis again

**Pull request merged**